__init__.py

```diff
@@ -0,0 +1,86 @@
+__author__ = 'Henry Senyondo'
+
+import os
+import platform
+import sys
+current_platform = platform.system().lower()
+if current_platform != 'windows':
+    pass
+current_platform = platform.system().lower()
+if current_platform != 'windows':
+    import pwd
+
+VERSION = 'v0.1'
+MASTER = True
```

Comment:
This line has been a source of a fair bit of pain in the retriever, so we should be rid of this and any associated branch dependencies.